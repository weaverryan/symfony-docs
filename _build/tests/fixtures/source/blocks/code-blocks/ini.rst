
.. code-block:: ini
    fetch = +refs/notes/*:refs/notes/*
