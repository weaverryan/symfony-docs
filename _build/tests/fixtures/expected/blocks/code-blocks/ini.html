<!DOCTYPE html>
<html>
<head>
    <meta charset="utf-8" />
</head>
<body>
<div class="literal-block">
    <div class="highlight-ini">
        <table class="highlighttable">
            <tr>
                <td class="linenos">
                    <div class="linenodiv">
                        <pre> 1</pre>
                    </div>
                </td>
                <td class="code">
                    <div class="highlight">
                        <pre class="hljs ini"><span class="hljs-attr">fetch</span> = +refs/notes/*:refs/notes/*</pre>
                    </div>
                </td>
            </tr>
        </table>
    </div>
</div>
</body>
</html>
